python/src/nanoarrow/array.py

```diff
@@ -490,3 +490,7 @@ def to_string(self, width_hint=80, items_hint=10) -> str:
 
     def __repr__(self) -> str:
         return self.to_string()
+
+    def inspect(self): # or dump?
```

Comment:
> This could probably fairly easily be extended to multiple chunks (by printing out in a loop?)

That's indeed true, that shouldn't be that hard. We don't need it right now, so would prefer to do it later